CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.18.8 (2024-11-27)
+
+
+
```

Comment:
There's nothing in the change log. Can we add a description of what was changed to help customers know why we did another release?